client/tests/test_config.py

```diff
@@ -1,30 +1,15 @@
 import os
+from unittest.mock import patch
 
 from securedrop_client.config import Config
 
 
-def test_missing_file(homedir):
+@patch.dict(os.environ, {"SD_SUBMISSION_KEY_FPR": ""})
+def test_missing_journalist_key_fpr():
     """
-    If a file doesn't exist, the config can still be created, but is "invalid".
+    If a key is missing, the config can still be loaded, but is "invalid".
```

Comment:
So...does anything actually use this `is_valid` property? A simple grep for it turns up nothing aside from these tests:
```
user@dev ~/g/f/securedrop-client (main)> rg is_valid
client/tests/test_config.py
16:    assert config.is_valid is False
30:    assert config.is_valid is False

client/securedrop_client/config.py
30:    def is_valid(self) -> bool:
```

Maybe we can just get rid of it?

Reply:
Done in 2e0ae4cd145b76a049740460f48c4264e1bb19d5.